### PR TITLE
Upgrade demoshop to version beta5

### PR DESCRIPTION
**Pull request by DavidOstrozlik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Keep demoshop upgraded with newest version of Framework
|New feature| No 
|BC breaks| No 
|Fixes issues| ...
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Review by boris-brtan — CHANGES_REQUESTED** [review]
Hi, THX for the upgrade, it could be in more commits but OK.
Please provide info in CHANGELOG.md let beta4 upgrade be your guide.

**Comment by boris-brtan** [comment]
are we able somehow to mention ELASTIC_SEARCH_INDEX_PREFIX='' ???

**Comment by DavidOstrozlik** [comment]
Hi, I added the info to CHANGELOG.md, but I don't think that ELASTIC_SEARCH_INDEX_PREFIX should be included there. I didn't do any changes related to it so I don't see the added value.

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,6 +41,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - Shopsys\FrameworkBundle\DataFixtures\Demo\MailTemplateDataFixture blocked from loading, because its overridden in ShopBundle 
     - Shopsys\FrameworkBundle\DataFixtures\Demo\MultidomainMailTemplateDataFixture blocked from loading, because its overridden in ShopBundle
     - fixed wrong links in mail template data fixtures
+- [#27 - Upgrade to shopsys framework 7.0.0-beta5](https://github.com/shopsys/demoshop/pull/33)
```
Comment (boris-brtan) [inline CHANGELOG.md]:
```suggestion
- [#33 - Upgrade demoshop to version beta5](https://github.com/shopsys/demoshop/pull/33)
```

**Review by boris-brtan — CHANGES_REQUESTED** [review]
Quick fix and it will be OK

**Review by boris-brtan — APPROVED** [review]

**Pull request merged**